data/processed/pre-process-repd.py

```diff
@@ -15,6 +15,7 @@ def clean_repd_csv(csv_str):
         csv_str = re.sub(rgx_match, '"', csv_str)
     return csv_str
 
+sys.stdin.reconfigure(encoding='iso-8859-1')
```

Comment:
@danstowell should I add this line back for the other pre-preprocess py files too?

Reply:
I'm not 100% sure - my opinion the ideal is to work in UTF-8 but explicitly handle non-unicode data as necessary, in other words "no, don't add it". I'd be pretty confident that OSM is in UTF-8, for example, so doesn't need reconfigure(). But if some of the other datasets are formatted in ascii rather than UTF-8, it would be needed there as well.